lib/ask/runtime/flow.ex

```diff
@@ -40,10 +40,20 @@ defmodule Ask.Runtime.Flow do
 
   defp next_step_by_skip_logic(flow, step, reply_value) do
     skip_logic =
-      step
-      |> Map.get("choices")
-      |> Enum.find(fn choice -> choice["value"] == reply_value end)
-      |> Map.get("skip_logic")
+      case step["type"] do
+        "numeric" ->
+          value = String.to_integer(reply_value)
+          step["ranges"]
```

Comment:
Is there a migration that adds this `ranges` property to existing numeric steps? If not, the runtime for existing steps will fail.